### PR TITLE
remove redundant spaces

### PR DESCRIPTION
**Pull request by sijad**
### Description

some keys had unnecessary spaces, that has been removed. it should fixes sijad/yaml-translation-utils#1

### Additional information

I've found this errors using https://codebeautify.org/yaml-validator, please check again before merging it.

### CODE REVIEW

File: locale/core.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ core:
     add_extension:
       developer_text: "Si vous êtes un développeur, vous pouvez <a>consultez la documentation</a> pour vous initier à créer vos propres extensions."
       install_text: "De même, vous pouvez rechercher des extensions sur la <a>communauté officielle de Flarum</a> et les installer manuellement en utilisant Composer."
-      temporary_text: "Prochainement, cette fenêtre vous permettra d’ajouter en quelques clics une extension à votre forum. Nous sommes d’ailleurs en train de développer cet écosystème !"
+      temporary_text: "Prochainement, cette fenêtre vous permettra d’ajouter en quelques clics une extension à votre forum. Nous sommes d’ailleurs en train de développer cet écosystème !"
```
Comment (sijad) [inline locale/core.yml]: I'm not sure what's wrong with this changed spaces either.

Comment (milescellar) [reply locale/core.yml]: Some spaces are nonbreaking spaces (it's a French rule, these spaces are before and/or after some characters, like ! ? : etc. They should stay like that.

Plus, this PR don't seems to fix the issue. Thanks anyway! 😎

Comment (sijad) [reply locale/core.yml]: yes, I've reverted those, but according to travis-ci that error gone

Comment (milescellar) [reply locale/core.yml]: Right! I'm not at home right now, so I can't merge it for now. Thanks a lot sijad!

**Comment by sijad** [comment]
happy to help, there's also another helper which checks both translations and yaml resource are identical and have same keys:

https://github.com/persianfla/flarum-ext-persian/blob/eba548897ce985d02159da9408713a429f748747/scripts/test.sh

you can add it to travis as an [additional script](https://github.com/persianfla/flarum-ext-persian/blob/eba548897ce985d02159da9408713a429f748747/.travis.yml#L6-L7) e.g:

```yaml
before_script:
  - go get github.com/sijad/yaml-translation-utils/yamllinter
  - go get github.com/sijad/yaml-translation-utils/same-yaml

script
  - bash ./scripts/linter.sh
  - bash ./scripts/test.sh
```

it's still beta but it'd be awesome to have your feedback.

**Comment by milescellar** [comment]
Thanks! 👍

**Pull request merged**